internal/build/docker_test.go

```diff
@@ -441,6 +465,19 @@ func (f *testFixture) assertFilesInImageWithContents(ref string, contents []path
 	}
 }
 
+func (f *testFixture) assertFileNotInImage(ref string, path string) {
```

Comment:
bluh it would be great if you could pass a single blob of expected info to a single `assertFiles` func and have a single `startContainer` call assert X files exist with contents Y, and Z files do not exist. because an additional `startContainer` call is non-trivial overhead and our test suite is already super slow as it is. (if this feels too finicky i'm happy to take it over)

Reply:
done!

Reply:
ah sorry now it's done! there were some merge issues.